### ISSUE TITLE
Render historic=* POI's

### ISSUE THREAD

**Issue by ZeiP** [gravitystorm/openstreetmap-carto#3518]
Tags historic=tank and historic=locomotive are currently not rendered, even though they are significant local landmarks that are also of public interest in the map. Could they, along with other missing historic=* POI's, be rendered either with some kind of generic historic POI icon or a more specific icon?

**Comment by Tomasz-W** [comment]
They have rather too low usage to consider dedicated rendering (tanks: 174, locomotives: 626). I would map them in combination with tourism=attraction + name=*, then they will be rendered at least as labels like in this museum: https://www.openstreetmap.org//#map=19/52.42047/16.93270

**Comment by ZeiP** [comment]
Is the usage count the only factor used in determining if a type of object should be rendered? These are usually quite visible and unique objects that are by nature low in count, but useful for both navigation and finding interesting POI's nearby. Not rendering them will probably mean that something else less specific is used instead. Could all of the currently unrendered historic=* values be rendered as monuments?

**Comment by kocio-pl** [comment]
The numbers are just indicators how likely is this tagging to be preferred one. Some numbers will be always low (for example there is not even 1000 countries in the world), but for others we try to wait until more people will invest their time to choose it.

**Comment by jeisenbe** [comment]
We also have a limited number of active contributors, versus a huge number
of database tags which could potentially be rendered.

**Comment by van-de-bugger** [comment]
> Even the most popular type of such objects, historic=aircraft, still hasn't passed the 2k limit, so I will close it for now, but please let us know if there is more of them.

There is no many aircrafts/locomotives/tanks because they are not rendered.

For example, in my city many tanks are tagged as `historic=memorial`. Mappers are result-oriented, they want to see objects on the map immediately, not in distant future.So mappers avoid using `historic=tank` because such POIs are invisible and useless. Most of the tanks are tagged as `historic=memorial` except few tanks added by me recently.

The same is about aircraft. Recently I changed `historic=memorial` to `historic=aircraft` and was very surprised because POI disappeared from the map.

The same is about locomotives. There is a railroad museum in my city with ~20 locomotives. All the locomotives tagged as `historic=memorial`, **not** `historic=locomotive`. Guess why.

So, if there too small number of locomotives/tanks/aircraft, please render them as some generic symbol (e. g. just a dot). In such a case number of mapped locomotives/tanks/aircraft will grow.

**Comment by Adamant36** [comment]
> or example, in my city many tanks are tagged as historic=memorial. Mappers are result-oriented, they want to see objects on the map immediately, not in distant future.

It seems as though historic=memorial is the only one out of those with a preset in iD Editor. My guess is that's a huge factor to. People aren't going to tag things with tags that aren't to add and that they don't know exist in the first place. So you should probably see if iD Editor can add them. Otherwise, they will never get used. Support there and in JSOM is a big factor in rendering support also, numbers or not. So without them supporting it, it probably wont be rendered whatever the usage numbers are.

**Comment by Ungoose** [comment]
A rendering for tank, cannon, and aircraft, (and probably locomotive) would be welcome indeed. There are many such objects in open-air museum exhibits which are regularly re-tagged as either historic=memorial or tourism=attraction (even though only some of them are) precisely because they are not visible when mapped correctly. And they are definitely of interest to at least some map users.
And yes, a preset in iD editor is also a major factor, but a preset alone would not advance the issue nearly as much without a rendering on carto. 
So, now that
> the 2k limit
is passed, maybe @kocio-pl will reconsider?

**Comment by kocio-pl** [comment]
For those which passed the limit, I'm OK with rendering.

**Comment by Ungoose** [comment]
Well, historic=aircraft is just over 2400 and historic=cannon is over 2200. Tanks are still under 500 and locomotives are just over 800, but I'd say rendering those would improve the mapping practices considerably. Even a simple dot will do, although having a nice tank/locomotive icon would be much closer to what people who map these features are trying to achieve. I personally would love to see historic=aircraft rendered as an icon currently used for aerodromes (maybe rotated 45 degrees to the side)
In any case, thanks for responding and reopening the issue. Can't wait to see all those open-air exhibits appear on the map!

**Comment by kocio-pl** [comment]
Is there anyone willing to prepare the code for those above the 2k threshold?

I guess it's better to have other icon for aircrafts, which looks more like some historic type.

**Comment by Ungoose** [comment]
Yes, I was thinking about it as well. However, many of the decommissioned aircraft in museums have jet and turboprop engines, so rendering them as WWI-style biplanes won't be representative either. Maybe something like the Airport icon from Maki would be a good middle ground.

**Comment by kocio-pl** [comment]
I think [Maki airfield icon](https://github.com/mapbox/maki/blob/master/icons/airfield-15.svg
) would be good enough after rescaling to 14px. The main problem seems to be somebody willing to do the coding.

**Comment by alexriabtsev** [comment]
> I think [Maki airfield icon](https://github.com/mapbox/maki/blob/master/icons/airfield-15.svg) would be good enough after rescaling to 14px. The main problem seems to be somebody willing to do the coding.

what kind of coding should be done?

**Comment by kocio-pl** [comment]
Look at the #3235 as the example what code changes are needed.

**Comment by alexriabtsev** [comment]
> Look at the #3235 as the example what code changes are needed.
OK, I'll do it

**Comment by Adamant36** [comment]
> historic=wreck seems to be even more popular (3134), but not mentioned here before.

It would be interesting to know what kind of objects it is used on, because a "wreck" is a pretty broad thing and might be hard to have a universal or understandable icon for something like that.

**Comment by marioxcc** [comment]
There are 1460 objects tagged with `historic=locomotive`. Add it to the render.

As said above, people will not tag objects if they are not displayed at all in the map. At least map them as a dot, until you make a specific icon.
